src/sentry/integrations/opsgenie/integration.py

```diff
@@ -51,8 +62,92 @@
 )
 
 
+class OpsgenieFinishedView(PipelineView):
+    def dispatch(self, request: Request, pipeline: Pipeline) -> Response:
+        response = pipeline.finish_pipeline()
+
+        integration = getattr(pipeline, "integration", None)
+        if not integration:
+            return response
+
+        messages.add_message(request, messages.SUCCESS, "Opsgenie integration installed.")
+
+        return self.redirect(
+            absolute_uri(
+                f"/settings/{pipeline.organization.slug}/integrations/opsgenie/{integration.id}/"
+            )
+        )
+
+
+class InstallationForm(forms.Form):
+    base_url = forms.CharField(
+        label=_("Base URL"),
+        help_text=_("Either https://api.opsgenie.com/ or https://api.eu.opsgenie.com/"),
+        widget=forms.TextInput(attrs={"placeholder": "https://api.opsgenie.com/"}),
+    )
+    api_key = forms.CharField(
+        label=("Opsgenie API Key"),
+        widget=forms.TextInput(
+            attrs={
+                "placeholder": _("5832fc6e14300a0d962240a8144466eef4ee93ef0d218477e55f11cf12fc3737")
```

Comment:
This seems like an odd placeholder to me assuming the key value will be different for each installation - is this what we do in other similar forms? If so it's chill like this, otherwise I'd recommend we follow their conventions